content/en/docs/setup/upgrade/notice/index.md

```diff
@@ -1,44 +1,27 @@
 ---
-title: 1.2 Upgrade Notice
-description: Important changes operators must understand before upgrading to Istio 1.2.
+title: 1.3 Upgrade Notice
+description: Important changes operators must understand before upgrading to Istio 1.3.
 weight: 5
 aliases:
     - /docs/setup/kubernetes/upgrade/notice/
 ---
 
-This page describes changes you need to be aware of when upgrading from
-Istio 1.1 to 1.2.  Here, we detail cases where we intentionally broke backwards
-compatibility.  We also mention cases where backwards compatibility was
-preserved but new behavior was introduced that would be surprising to someone
-familiar with the use and operation of Istio 1.1.
+This page describes changes you need to be aware of when upgrading from Istio 1.2 to 1.3. Here, we
+detail cases where we intentionally broke backwards compatibility.
 
-For an overview of new features introduced with Istio 1.2, please refer
-to the [1.2 release notes](/about/notes/1.2/).
+For an overview of new features introduced with Istio 1.3, please refer to the [1.3 release notes](/about/notes/1.3/).
 
-## Installation and Upgrade
+## Trust Domain Validation
 
-{{< tip >}}
-The configuration model for Mixer has been simplified. Support for
-adapter-specific and template-specific Custom Resources has been
-removed by default in 1.2 and will be removed entirely in 1.3.
-Please move to the new configuration model.
-{{< /tip >}}
+When mutual TLS is enabled on a proxy using authentication policy, the proxy will reject a request
+from client if the trust domain extracted from the client certificate is not the same to the trust
+domain extracted from the proxy's own certificate. This is called trust domain validation which is
+new in Istio 1.3.
```

Comment:
Can we include the rest of the information in PR #4931? I would suggest some edits too:
```
{{< gloss >}}Trust domain validation{{< /gloss >}} is new in Istio 1.3. If you only have one trust domain or not using mutal TLS from an authentication policy, there is nothing you must do.
```